app/services/coronavirus_pages/content_builder.rb

```diff
@@ -97,4 +98,22 @@ def add_live_stream(data)
       data["live_stream"] = github_live_stream_data.merge(persisted_live_stream_data)
     end
   end
+
+  def hidden_search_terms
+    sections = sub_sections_data.map do |section|
+      [section[:title], subsections(section[:sub_sections])]
+    end
+
+    sections.flatten.compact
```

Comment:
I see you're using compact to remove any nil values, are you at all concerned about empty strings? I wonder if you might want select(&:present?).

Also would you want repeated terms multiple times (I see "Popular content" from your example) or would you want to `.uniq` them so they don't distort relevancy?

Reply:
Thanks! I forgot about uniqueness.

I’ve never used `select(&:present?)` before. It doesn’t read as nicely as `compact` but is definitely more useful.